setup.py

```diff
@@ -160,7 +160,7 @@ def get_exts_for(name):
             "Programming Language :: Python :: 3.8",
         ],
         use_scm_version=True,
```

Comment:
```suggestion
        use_scm_version={
            "version_scheme":  "release-branch-semver",
        }
```

> Many tools, especially those that invoke setup.py for any reason, may continue to rely on setup_requires. For maximum compatibility with those uses, consider also including a setup_requires directive (described below in setup.py usage and setup.cfg).

We probably still want to have it in setup.py as well.

Reply:
Right but setuptools_scm is the only one reading this, and it'll read it from the toml file, so we're good.